react-commenting.js

```diff
@@ -31,20 +31,20 @@ class Board extends Component{
   }
 
   handleRoll = () => {
-    // 3)
+    // 3)you are destructuring rollImages and diceLog
     let { rollImages, diceLog } = this.state
     let randomNum = Math.ceil(Math.random() * rollImages.length)
-    // 4)
+    // 4)creating a new variable newRoll to equal rollImages which is an array that has the built in method math.random giving the index a random number  1 through 6
     let newRoll = rollImages[randomNum]
-    // 5)
+    // 5)this.setState updates the currentPic everytime time and diceLog is logging every roll
     this.setState({ currentPic: newRoll, diceLog: [... diceLog, newRoll] })
   }
 
   render(){
     const { currentPic, diceLog } = this.state
     return(
       <div id="board-container">
-        // 6)
+        // 6)your passing the Dice component to the parent class
```

Comment:
this is more that the parent class is rendering `Dice` and sending it `roll` and `currentPic` as props